app/views/foods/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="m-20">
+  <h1 class="mb-11 text-3xl font-medium">Foods</h1>
+  <div id="foodsList">
+  <div class="w-full flex justify-end">
+    <button class="bg-blue-500 hover:bg-blue-400 text-white font-bold py-2 px-4 border-b-4 border-blue-700 hover:border-blue-500 rounded">
+      <%= link_to "add new food",  new_user_recipe_food_path(params[:user_id],params[:recipe_id])%>
+    </button>
+  </div>
+  <table class="mb-20 border-2 text-left w-[80vw] border-slate-700 text-xs text-gray-700 uppercase bg-gray-50 dark:bg-gray-700 dark:text-gray-400 p-4">
+      <tr>
+        <th class="text-xs text-white uppercase bg-gray-700 p-4">Food</th>
+        <th class="text-xs text-white uppercase bg-gray-700 p-4">Measurement unit</th>
+        <th class="text-xs text-white uppercase bg-gray-700 p-4">Price</th>
+        <th class="text-xs text-white uppercase bg-gray-700 p-4">Action</th>
+      </tr>
+      <% @foods.each do |food| %>
+        <%= render food %>
+         <% if user_signed_in? %>
+                  <th>
+                    <%= link_to "Remove",user_recipe_food_path(params[:user_id], params[:recipe_id], food.id),data: {turbo_method: :delete}, class:"p-4 font-medium hover:text-red-500"  %>
+                    <%= link_to "Modify",edit_user_recipe_path(params[:user_id], params[:recipe_id], food.id), method: :edit, class:"p-4 font-medium hover:text-green-400"  %>
+                  </th>
+</tr>
+            <% end %>
+      <% end %>
+
+      </table>
```

Comment:
- [ ] kindly ensure your design matches the design given, the food table should be below the add food button check the comparison below

Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/218186320-afc5c8d9-5c80-4cc1-a108-2f93e952914b.png)  |  ![image](https://user-images.githubusercontent.com/61408860/218186401-ee13d0ac-6afa-45b3-a513-6b3c1e2dab8e.png)